src/apps/frontend/services/account.service.ts

```diff
@@ -9,7 +9,7 @@ export default class AccountService extends APIService {
       localStorage.getItem('access-token'),
     ) as AccessToken;
 
-    return this.apiClient.get(`/accounts/${userAccessToken.accountId}`, {
+    return this.apiClient.get(`/account/${userAccessToken.accountId}`, {
```

Comment:
Resource should always be plural in URI, please revert this change

Reply:
Reverted and changed the endpoint from `account` to `accounts`